styles/main.css

```diff
@@ -1,22 +1,60 @@
 @CHARSET "UTF-8";
-<<<<<<< HEAD
 
-body { background-color: aqua; font-family: 'Domine', serif;}
-h1 { color: grey; background-image: url(../images/gradient.png); background-repeat: repeat-y; font-family: 'Domine', serif;}
-h2 { color: grey; background-image: url(../images/gradient.png); background-repeat: repeat-y; font-family: 'Roboto', sans-serif;}
-a { color: gold; text-decoration:none; }
-ul { color: purple; list-style-type: square;}
-li { color: purple;}
+body { 
+    background-color: aqua; 
+    font-family: 'Domine', serif;
+}
+h1 { 
+    color: grey; 
+    background-image: url(../images/gradient.png); 
+    background-repeat: repeat-x; 
+    font-family: 'Domine', serif;
+}
+h2 { 
+    color: grey; 
+    background-image: url(../images/gradient.png); 
+    background-repeat: repeat-x; 
+    font-family: 'Roboto', sans-serif;
+}
+a { 
+    color: gold; 
+    text-decoration:none; 
+}
+ul { 
+    color: purple; 
+    list-style-type: square;
+}
+li { 
+    color: purple;
+}
 
-.byline { color:rgba(0,0,0,0.5); font-size:"70%";}
-=======
-body { background-color: aqua;}
-h1, h2 { color: grey;}
-a { color: gold;}
-header nav a { color: purple;}
-.byline { color:rgba(0,0,0,0.5)}
-#page-copyright { color: red; text-transform: uppercase; letter-spacing: 3px;}
->>>>>>> refs/heads/master
-.diffcolor { background-color: blue;}
-.bio { text-indent: 3em; text-align: center; font-size: 12px;}
-.headerlink {list-style-type: none;}
\ No newline at end of file
+.byline { 
+    color:rgba(0,0,0,0.5); 
+    font-size: 70%;
+}
+
+header nav a { 
+    color: purple;
+}
+header nav ul {
+    list-style-type: none;
+}
+
+#page-copyright { 
+    color: red; 
+    text-transform: uppercase; 
+    letter-spacing: 3px;
+}
+
+.diffcolor { 
+    background-color: blue;
+}
+
+.bio p { 
+    text-indent: 3em; 
+}
+
+footer {
+    text-align: center; 
+    font-size: 12px;
+}

```

Comment:
You had these properties in the `.bio` rule, but they need to be here in the footer rule